configs/ti_release_am62x_sk_defconfig

```diff
@@ -39,6 +39,7 @@ BR2_TARGET_OPTEE_OS_CUSTOM_REPO_URL="https://github.com/OP-TEE/optee_os.git"
 BR2_TARGET_OPTEE_OS_CUSTOM_REPO_VERSION="8f645256efc0dc66bd5c118778b0b50c44469ae1"
 BR2_TARGET_OPTEE_OS_NEEDS_PYTHON_CRYPTOGRAPHY=y
 BR2_TARGET_OPTEE_OS_PLATFORM="k3-am62x"
+BR2_TARGET_OPTEE_OS_ADDITIONAL_VARIABLES="CFG_WITH_SOFTWARE_PRNG=y CFG_TEE_CORE_LOG_LEVEL=1"
```

Comment:
In Debian and Arago, we have only `CFG_TEE_CORE_LOG_LEVEL` for AM62x. `CFG_WITH_SOFTWARE_PRNG` is an argument only for AM62px.